fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -62,8 +62,8 @@ def take_off
         # Loading any .env files before any lanes are called since
         # variables like FASTLANE_HIDE_CHANGELOG and FASTLANE_DISABLE_COLORS
         # need to be set early on in execution
-        require 'fastlane/helper/dotenv_helper'
-        Fastlane::Helper::DotenvHelper.load_dot_env(nil)
+        # TODO: Should load_dot_env be moved to the top of take_off in order to support FASTLANE_DISABLE_ANIMATION preference?
+        load_dot_env
```

Comment:
`load_dot_env` will internally call `require 'fastlane/helper/dotenv_helper'`

If I were to move this call up to the top of `take_off`, this would mean that we would require `fastlane/helper/dotenv_helper` before the long running `require 'fastlane'` call... Would that cause a problem? If so, I guess I should leave this call where it is.

Reply:
I _think_ this needs to stay after `require 'fastlane'` 🤔 Trial/error might prove me wrong though 🙃

Reply:
Thanks for the info.. Originally I was worried that I wouldn't be able to tell if it worked or not, but it was pretty obvious in the end 😂 

If I try to move it above `require 'fastlane'` the method completely explodes so thats a pretty good sign that it's a no-go 😅 

I have however opted to move it above the locale check and `print_bundle_exec_warning` so that `LANG`, `LC_ALL` and `SKIP_SLOW_FASTLANE_WARNING` can be defined in external .env files. 

My understanding from the [docs](https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables) is that UTF8 is required by things like xcodebuild and not fastlane/ruby itself so specifying something like `LANG` in a .env file should be fine? I have to define these variables on my Jenkins CI setup already so it'll be ace if I can move them into my **.env.jenkins** file with this change.